runtime/Go/antlr/interval_set.go

```diff
@@ -23,7 +23,15 @@ func NewInterval(start, stop int) *Interval {
 	return i
 }
 
-func (i *Interval) contains(item int) bool {
+func (i *Interval) GetStart() int {
```

Comment:
Go style doesn't use the word Get in getters.  This would just be Start().

Looking at the implementation of this class in Java I see that the start/stop are just called a and b (and are public and not final).  Since we aren't using subclasses for this class we could probably follow suit and expose these values directly.

http://www.antlr.org/api/Java/org/antlr/v4/runtime/misc/Interval.html